dotenv.cabal

```diff
@@ -1,5 +1,5 @@
 name:                dotenv
-version:             0.11.0.0
+version:             0.11.0.1
```

Comment:
@CristhianMotoche based on the fact we are moving some modules from the `exposed-modules` section to the `other-modules` section, I think we should publish this change as a major release since it would break some builds if another project imports directly one of the modules previously exposed

Reply:
Hey @sestrella Mmm I didn't move any module away from `exposed-modules`. My only change was to add `Configuration.Dotenv.Internal` as a new exposed module. For that reason, I think this is a minor change. WDYT?

Reply:
@CristhianMotoche my mistake, I misread the diff, you are correct, it appears that we are only adding a new module to the `exposed-modules` section, in this case, a minor release appears to be the best option.